src/meta/processors/indexMan/CreateEdgeIndexProcessor.cpp

```diff
@@ -126,13 +127,28 @@ void CreateEdgeIndexProcessor::process(const cpp2::CreateEdgeIndexReq& req) {
             return;
         }
         cpp2::ColumnDef col = *iter;
-        if (col.type.get_type() == meta::cpp2::PropertyType::STRING) {
+        if (col.type.get_type() == meta::cpp2::PropertyType::FIXED_STRING) {
+            if (*col.type.get_type_length() > MAX_INDEX_TYPE_LENGTH) {
+                LOG(ERROR) << "Unsupport index type lengths greater than "
+                           << MAX_INDEX_TYPE_LENGTH << " : "
+                           << field.get_name();
```

Comment:
handle  `handleErrorCode  `and  `onFinished`

Reply:
> handle `handleErrorCode `and `onFinished`

Good point. thanks. fixed.